benchmark/_interfaces.py

```diff
@@ -87,3 +87,25 @@ def measure(f, *a, **kw):
         :return: Deferred firing when measurement has been taken, with the
             value of the measurement.
         """
+
+
```

Comment:
Two new interfaces, the common stuff an scenario needs: a setup, and a request generator that will make a request.